workspaces/cvat/template.yaml

```diff
@@ -1,4 +1,10 @@
-# Docker containers that are part of the Workspace
+# Workspace arguments
+arguments:
+  parameters:
+  - name: storage-prefix
+    displayName: Directory in default object storage
+    value: workflow-data
```

Comment:
@rushtehrani this needs to be `<namespace>/workflow-data`.

Reply:
We need add a `hint` that says "This prefix is relative to current namespace". And then add `{{workspace.namespace}}` to [this line](https://github.com/onepanelio/templates/pull/18/files/db2399e2217ad6c31a15341c3117d1c1e2ca3ca8#diff-e0617e049385ef6221f29ac9267b6e53R75). Since this is fully automated, we can be more opinionated around how files are stored but the user can always clone the Workflows and Workspaces and configure it to their needs.

Reply:
ok thanks